Dockerfile

```diff
@@ -1 +1,51 @@
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/centos/base-image:java8
+ENV PENTAHO_CE_VERSION=8.1.0.0-365
+ENV Mysql_Connector_Version=8.0.24
```

Comment:
Each `ENV` will create a new Docker layer. Please concatenate them (see ark_activemq for example).

Reply:
Changed to single ENV, But I think ENV it will not create new docker layer , Please correct me if I am wrong.